<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="fromStationLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="toStationLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>yyyy-MM-dd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fetchButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>711</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
